src/ngx_http_lua_ssl_session_storeby.c

```diff
@@ -252,6 +252,11 @@ ngx_http_lua_ssl_sess_store_handler(ngx_ssl_conn_t *ssl_conn,
     cctx->session_id.data = sess->session_id;
     cctx->session_id.len = sess->session_id_length;
     cctx->done = 0;
+#if (NGX_HTTP_V2)
```

Comment:
I am not quite sure why storing session will touch http2, alpn and session resumption are orthogonal.

Reply:
Yeah, this is definitely wrong. session storing happens *after* a full SSL handshake and thus also ALPN selection and ssl set_cert callback.